bundles/org.openhab.binding.souliss/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,942 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="souliss"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+
+	<bridge-type id="gateway">
+		<label>Gateway</label>
+		<description>Represents a Souliss Gateway.</description>
+
+		<config-description>
+			<parameter name="GATEWAY_IP_ADDRESS" type="text" required="true">
+				<label>IP or Host Name</label>
+				<description>Will be resolved by discovery if auto configured
+				</description>
+			</parameter>
+			<parameter name="GATEWAY_PORT_NUMBER" type="integer" required="true">
+				<label>Gateway port</label>
+				<description>Default is 230 UDP.
+				</description>
+				<default>230</default>
+			</parameter>
+			<parameter name="PREFERRED_LOCAL_PORT_NUMBER" type="integer" required="true">
+				<label>Preferred local port number</label>
```

Comment:
```suggestion
				<label>Preferred Local Port Number</label>
```

Apply capitals to all words in the label unless they are joining words like 'a' or 'or'